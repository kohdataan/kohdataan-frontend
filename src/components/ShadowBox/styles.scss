@import '../../styles/colors.scss';

.shadow-box-container {
  border-radius: 10px;
  box-shadow: 0 3px 15px -2px rgba(0, 0, 0, 0.3);
  margin: 20px;
}
