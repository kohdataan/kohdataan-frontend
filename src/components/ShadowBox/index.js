import React, { memo } from 'react'
import propTypes from 'prop-types'
import './styles.scss'

const ShadowBox = props => {
  const { children } = props
  return <div className="shadow-box-container">{children}</div>
}

ShadowBox.propTypes = {
  children: propTypes.oneOfType([
    propTypes.string,
    propTypes.instanceOf(Array),
    propTypes.node,
  ]).isRequired,
}

export default memo(ShadowBox)
