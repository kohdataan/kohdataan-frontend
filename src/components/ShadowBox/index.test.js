import React from 'react'
import { create } from 'react-test-renderer'
import ShadowBox from './index'

describe('Shadowbox', () => {
  test('it matches the snapshot', () => {
    const component = create(
      <ShadowBox>
        <p>Test</p>
      </ShadowBox>
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
