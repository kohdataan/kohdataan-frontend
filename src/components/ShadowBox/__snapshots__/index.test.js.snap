// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shadowbox it matches the snapshot 1`] = `
<div
  className="shadow-box-container"
>
  <p>
    Test
  </p>
</div>
`;
